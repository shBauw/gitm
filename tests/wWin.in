place A1
place A2
place B1
place B2
place C1
place C2
place F1
place D2
place E1
place E2
