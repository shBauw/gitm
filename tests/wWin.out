White wins!
A1A2B1B2C1C2F1D2E1E2
Thank you for playing!
